lib/helpers/setup.js

```diff
@@ -17,25 +16,32 @@ var requireFromCLI = require('./require-from-cli');
   @param {Number} [options.timeout=20000]
   @param {Object} [options.tmpenv]
   @param {String} [options.cliPath='ember-cli'] path to the `ember-cli` dependency
+  @param {Boolean} [options.disableMocking=false] turn on the real npm (and bower) to test
 */
 module.exports = function setupTestHooks(scope, options) {
   var timeout = options && options.timeout || 20000;
   var tmp = options && options.tmpenv || scope.tmpenv || tmpenv;
+  var disableMocking = options && options.disableMocking;
+
   scope.timeout(timeout);
 
-  if (options && options.cliPath) {
-    requireFromCLI.setBasePath(options.cliPath);
-  }
+  if (!disableMocking) {
+    var requireFromCLI = require('./require-from-cli');
 
-  var Blueprint = requireFromCLI('lib/models/blueprint');
+    if (options && options.cliPath) {
+      requireFromCLI.setBasePath(options.cliPath);
+    }
```

Comment:
I think we should probably `setBasePath()` regardless of whether we are mocking or not